states/afk/templates/routing_policy/eos/statement.j2

```diff
@@ -83,7 +83,11 @@ route-map {{ route_map_name }} {{ ACTION[actions["config"]["policy-result"]] }}
 {% endif %}
 
 {% if actions["bgp-actions"] and actions["bgp-actions"]["config"].get("set-next-hop") %}
+  {% if ":" in ipaddress %}
```

Comment:
Can we use an address family test here?
ip_address(ipadress).version == 6

Reply:
There is no such helper in Jinja2.